### PR TITLE
add property to keep track of which spherical harmonic is attached to…

### PR DESCRIPTION
**Pull request by blondegeek**
… which radial function

### CODE REVIEW

File: e3nn/kernel.py [hunk 1]
```diff
@@ -57,6 +58,8 @@ def lm_normalization(l_out, l_in):
             for mul_in, l_in, p_in in self.Rs_in:
                 l_filters = self.get_l_filters(l_in, p_in, l_out, p_out)
                 num_summed_elements += mul_in * len(l_filters)
+                list_of_l_filters += [l for l in l_filters for i in range
```
Comment (mariogeiger) [inline e3nn/kernel.py]: don't you want to save more information? like a tuple `(l_in, p_in, l_out, p_out, l)`?

**Comment by blondegeek** [comment]
My motivation for adding this list is to facilitate plotting kernels. I'm happy for the list to store the full information, but I only need l_filter. See [this in progress function](https://github.com/blondegeek/e3nn_tutorial/blob/volume/spherical.py#L207) (that I hope to add as a method to Kernel) that I'm using to visualize the kernel and [this notebook](https://github.com/blondegeek/e3nn_tutorial/blob/volume/plot_filter.ipynb) for an example of how to use it.

File: e3nn/kernel.py (hunk 1 above)
Comment (blondegeek) [reply e3nn/kernel.py]: My motivation for adding this list is to facilitate plotting kernels. I'm happy for the list to store the full information, but I only need l_filter. See [this in progress function](https://github.com/blondegeek/e3nn_tutorial/blob/volume/spherical.py#L207) (that I hope to add as a method to Kernel) that I'm using to visualize the kernel and [this notebook](https://github.com/blondegeek/e3nn_tutorial/blob/volume/plot_filter.ipynb) for an example of how to use it.

Comment (mariogeiger) [reply e3nn/kernel.py]: As you wish, I merge!

**Pull request merged**